website/docs/docs/cloud/manage-access/scim.md

```diff
@@ -27,32 +41,60 @@ If your IdP isn’t on the list, it can be supported using dbt Cloud APIs (_docs
 
 Please complete the [setup SSO with Okta](/docs/cloud/manage-access/set-up-sso-okta) steps before configuring SCIM settings.
 
-To configure the SCIM settings for Okta:
+
+### Set up dbt Cloud
+
+To retrieve the necessary dbt Cloud configurations for use in Okta:
 
 1. Navigate to your dbt Cloud **Account settings**.
 2. Select **Single sign-on** from the left-side menu.
-3. Click **Edit**, scroll to the bottom of your Okta configuration settings, and click **Enable SCIM**.
-    - In later steps, you will need both the **SCIM base URL** and **SCIM token**.
+3. Scroll to the bottom of your Okta configuration settings and click **Enable SCIM**.
+    <Lightbox src="/img/docs/dbt-cloud/access-control/enable-scim.png" width="60%" title="SCIM enabled in the Okta configuration settings." />
```

Comment:
just double checking if this is a real account number in the screenshot?

Reply:
Yes but no concerns with exposing. Wanted unredacted screenshot for the workflow